RELEASES.md

```diff
@@ -11,6 +11,10 @@
 - Added `child_count`  method to `LayoutTree` for querying the number of children of a node. Required because the `children` method now returns an iterator instead of an array.
 - Added `is_childless` method to `LayoutTree` for querying whether a node has no children.
 
+### Fixes
+
+- `min_size` now overrides `max_size` which overrides `size` #261
```

Comment:
This being a regression, I feel like this ought to go into a 0.2.1 release (unless we're planning to release 0.3.0 imminently).

Reply:
I think I would agree that perhaps this should be a 0.2.1 update. However I've also rebased this off of main, which includes changes set for 0.3.0 (currently). What would be the right process for having this in a 0.2.1 release while not including changes set for a later release? Maybe this is more of a gh question.

Reply:
>  What would be the right process for having this in a 0.2.1 release while not including changes set for a later release?

I've created a `0.2.x` branch. You can rebase on top of that, and set it as the base branch for the pull request.